open_humans/settings.py

```diff
@@ -355,7 +355,7 @@ def __unicode__(self):
 ACCOUNT_LOGIN_REDIRECT_URL = LOGIN_REDIRECT_URL
 ACCOUNT_OPEN_SIGNUP = to_bool('ACCOUNT_OPEN_SIGNUP', 'true')
 ACCOUNT_PASSWORD_MIN_LEN = 8
-ACCOUNT_SIGNUP_REDIRECT_URL = 'my-member-signup-setup-1'
+ACCOUNT_SIGNUP_REDIRECT_URL = 'my-member-signup-setup-2'
```

Comment:
I think it's fine to remove this extra step as well and default redirect to 'welcome' after login. Notably, the welcome pages now have `/welcome/profile/`.